field/field.go

```diff
@@ -56,6 +56,28 @@ func (f Static) HasPrefix(prefix Static) bool {
 	return len(f) >= len(prefix) && prefix.Equal(f[:len(prefix)])
 }
 
+func (f Static) IsPrefixOf(of Static) bool {
+	return of.HasPrefix(f)
+}
+
```

Comment:
`HasPrefix` is sufficient, so I wouldn't add this.

Reply:
It's not used so I will delete it but I found it useful at one point for code clarity at a call site.